pagopa/io-app
# feat: [IOBP-224] IDPay discount initiative payment method

## ⚠️ This PR depends on #4982 and #4994

## Short description
This PR adds an initial page to start the new flow for the discount initiative to choose by which method the user wants to authorize payments.

## List of changes proposed in this pull request
- Added a new screen `IdPayDiscountInstrumentsScreen`
- Added and setup the use of saga for the configuration folder which actually it handles only those async request of this screen;
- Added a new ad-hoc component `IdPayDiscountInstrumentEnrollmentSwitch` that handles the instrument given as input and manage the UI to show based on if the instrument is `IDPAYCODE` or `QRCODE` and the switch that navigates to the onboarding screen
- Integrated a bottom sheet to show how to pay with CIE;

## How to test
Open a discount initiative details and press on "Metodi di pagamento attivi", then you should be able to enable/disable the CIE as payment method.

## Preview


https://github.com/pagopa/io-app/assets/34343582/2625d95e-4aa6-4815-9bcb-2d9a8e91ef36
